### PR TITLE
Update getting-started guides to use operator by default

### PR DESCRIPTION
**Pull request by caseydavenport**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
Calico install documentation now uses operator-based instructions by default
```

### CODE REVIEW

File: calico/getting-started/kubernetes/installation/config-options.md [hunk 1]
```diff
@@ -1,10 +1,81 @@
...
+
+#### {{site.prodname}} operator
+
+{{site.prodname}} is installed by an operator which manages the installation, upgrade, and general lifecycle of a {{site.prodname}} cluster. The operator is
+installed directly on the cluster as a Deployment, and is configured through one or more custom Kubernetes API resources.
+
+#### {{site.prodname}} manifests
+
+{{site.prodname}} can also be installed using raw manifests as an alternative to the operator which contain the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. Using manifests is not recommended as they cannot automatically manage the lifecycle of the {{site.prodname}} as the operator does. However, manifests may be useful for clusters which require highly specific modifications to the underlying Kubernetes resources.
+
```
Comment (Akasurde) [inline calico/getting-started/kubernetes/installation/config-options.md:21]:
```suggestion
{{site.prodname}} can also be installed using raw manifests as an alternative to the operator which contains the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. Using manifests is not recommended as they cannot automatically manage the lifecycle of the {{site.prodname}} as the operator does. However, manifests may be useful for clusters that require highly specific modifications to the underlying Kubernetes resources.

```

File: calico/getting-started/kubernetes/installation/config-options.md [hunk 2]
```diff
@@ -1,10 +1,81 @@
...
+#### About customizing an operator install
+
+Operator installations read their configuration from a specifc set of Kubernetes APIs. This APIs are installed on the cluster
+as part of `tigera-operator.yaml` in the `operator.tigera.io/v1` API group.
+
+- [Installation]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.Installation): a singleton resource with name "default" that
+  configures common installation parameters for a {{site.prodname}} cluster.
+- [APIServer]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.Installation): a singleton resource with name "default" that
+  configures installation of the {{site.prodname}} API server extension.
```
Comment (Akasurde) [inline calico/getting-started/kubernetes/installation/config-options.md:36]:
```suggestion
Operator installations read their configuration from a specific set of Kubernetes APIs. These APIs are installed on the cluster
as part of `tigera-operator.yaml` in the `operator.tigera.io/v1` API group.

- [Installation]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.Installation): a singleton resource with name "default" that
  configures common installation parameters for a {{site.prodname}} cluster.
- [APIServer]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.Installation): a singleton resource with name "default" that
  configures installation of the {{site.prodname}} API server extension.
```

File: calico/getting-started/kubernetes/self-managed-onprem/onpremises.md [hunk 3]
```diff
@@ -11,18 +11,16 @@ Install {{site.prodname}} to provide both networking and network policy for self
...
+#### {{site.prodname}} operator
 
-### Concepts
+{{site.prodname}} is installed by an operator which manages the installation, upgrade, and general lifecycle of a {{site.prodname}} cluster. The operator is
+installed directly on the cluster as a Deployment, and is configured through one or more custom Kubernetes API resources.
 
 #### {{site.prodname}} manifests
 
-{{site.prodname}} provides manifests for easy customization. Each manifest contains the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. You may want to [customize the {{site.prodname}} manifests]({{site.baseurl}}/getting-started/kubernetes/installation/config-options) before installing {{site.prodname}} on nodes.
+{{site.prodname}} can also be installed using raw manifests as an alternative to the operator which contain the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. Using manifests is not recommended as they cannot automatically manage the lifecycle of the {{site.prodname}} as the operator does. However, manifests may be useful for clusters which require highly specific modifications to the underlying Kubernetes resources.
```
Comment (Akasurde) [inline calico/getting-started/kubernetes/self-managed-onprem/onpremises.md:23]:
```suggestion
{{site.prodname}} can also be installed using raw manifests as an alternative to the operator which contains the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. Using manifests is not recommended as they cannot automatically manage the lifecycle of the {{site.prodname}} as the operator does. However, manifests may be useful for clusters that require highly specific modifications to the underlying Kubernetes resources.
```

File: calico/getting-started/kubernetes/self-managed-public-cloud/gce.md [hunk 4]
```diff
@@ -160,16 +160,22 @@ worker-2     NotReady   <none>   5s      v1.17.2
...
-On the controller, install {{site.prodname}} from the manifest:
+On the controller, install {{site.prodname}} using the operator:
 
 ```
-curl {{ "/manifests/calico.yaml" | absolute_url }} -O
+kubectl create -f {{ "/manifests/tigera-operator.yaml" | absolute_url }}
 ```
 
-If you wish to customize the {{site.prodname}} install, customize the downloaded calico.yaml manifest.  Then apply the manifest to install {{site.prodname}}.
+Downlaod the custom resources necessary to configure {{site.prodname}}
```
Comment (bcreane) [inline calico/getting-started/kubernetes/self-managed-public-cloud/gce.md:169]: typo - Download

**Review by bcreane — APPROVED** [review]
looks good, just the one typo

File: calico/getting-started/kubernetes/self-managed-onprem/onpremises.md [hunk 5]
```diff
@@ -11,18 +11,16 @@ Install {{site.prodname}} to provide both networking and network policy for self
...
 
-### Concepts
+{{site.prodname}} is installed by an operator which manages the installation, upgrade, and general lifecycle of a {{site.prodname}} cluster. The operator is
+installed directly on the cluster as a Deployment, and is configured through one or more custom Kubernetes API resources.
 
 #### {{site.prodname}} manifests
 
-{{site.prodname}} provides manifests for easy customization. Each manifest contains the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. You may want to [customize the {{site.prodname}} manifests]({{site.baseurl}}/getting-started/kubernetes/installation/config-options) before installing {{site.prodname}} on nodes.
+{{site.prodname}} can also be installed using raw manifests as an alternative to the operator. The manifests contain the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. Using manifests is not recommended as they cannot automatically manage the lifecycle of the {{site.prodname}} as the operator does. However, manifests may be useful for clusters that require highly specific modifications to the underlying Kubernetes resources.
```
Comment (lwr20) [inline calico/getting-started/kubernetes/self-managed-onprem/onpremises.md:23]: Maybe also mention that at upgrade time, someone will need to repeat the customisation process on the uplevel manifest, whereas with Operator that's all taken care of.

**Review by lwr20 — APPROVED** [review]

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Pull request merged**